SECURITY.md

```diff
@@ -0,0 +1,211 @@
+# Security Release Process
+
+Rook is a large growing community of storage providers, contributors, and users. The Rook community
+has adopted this security disclosure and response policy to ensure we responsibly handle critical
+issues.
+
+This policy is adapted from the policies of the following CNCF projects:
+
+* [Envoy](https://github.com/envoyproxy/envoy)
+* [CoreDNS](https://github.com/coredns/coredns)
+
+The Rook community credits and appreciates the example and security best practices that they have
+published openly.
+
+## Security Audit
+
+A third party security audit was performed in December 2019 by [Trail of
+Bits](https://www.trailofbits.com/). The full security report has been published and is [available for
+download](https://drive.google.com/file/d/1rOwrwYmBUpLUm6W5J5rhXvdVit818hWJ/view?usp=sharing).
+
+## Product Security Team (PST)
+
+Security vulnerabilities should be handled quickly and sometimes privately. The primary goal of this
+process is to reduce the total time users are vulnerable to publicly known exploits.
+
+The Product Security Team (PST) is responsible for organizing the entire response including internal
+communication and external disclosure.
+
+The initial Product Security Team will consist of the set of maintainers that volunteered. Every
+beta or stable storage provider **MUST** have a representative on the PST.
+
+### Mailing lists
+
+* [security@rook.io](mailto:security@rook.io): for any security concerns. Received by Product
+  Security Team members, and used by this Team to discuss security issues and fixes.
+* rook-distributors-announce@lists.cncf.io: for early private information on Security patch
```

Comment:
Please make this a `mailto` link as well.

Reply:
made this `mailto` link change